package.json

```diff
@@ -5,8 +5,8 @@
   "config": {
     "webservice_version": "1.15.0",
     "use_circle": true,
-    "circle_ci_source": "https://app.circleci.com/pipelines/github/dockstore/dockstore/10673/workflows/e44fc131-9382-4f1f-b8b3-666468eb7384/jobs/39289/artifacts",
-    "circle_build_id": "39289"
+    "circle_ci_source": "https://app.circleci.com/pipelines/github/dockstore/dockstore/10696/workflows/4406a2db-e982-45c0-84be-ef925a4ac7b0/jobs/39419/artifacts",
+    "circle_build_id": "39419"
```

Comment:
I will point this at the develop artifacts after the companion webservice PR merges.